### PR TITLE
[CIRCSTORE-140] Implementing an endpoint for searching scheduled notices in groups

### PR DESCRIPTION
**Pull request by dmytrotur**
## Purpose
Implement an endpoint for searching scheduled notices in groups
Resolves: [CIRCSTORE-140](https://issues.folio.org/browse/CIRCSTORE-140)

Based on changes form https://github.com/folio-org/mod-circulation-storage/pull/186

### CODE REVIEW

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -296,7 +296,7 @@
     },
     {
       "id": "scheduled-notice-storage",
-      "version": "0.1",
+      "version": "0.3",
```
Comment (kostyantyn-kh) [inline descriptors/ModuleDescriptor-template.json]: Should it be "0.2" instead of "0.3"?

File: ramls/patron-notice-policy.json [hunk 2]
```diff
@@ -344,6 +317,7 @@
                   "Recall request",
                   "Recall loanee",
                   "Hold request",
+                  "Request Expiration",
```
Comment (kostyantyn-kh) [inline ramls/patron-notice-policy.json]:
```suggestion
                  "Request expiration",
```

File: ramls/scheduled-notice-triggering-event.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "string",
+  "description": "Scheduled notice triggering event",
+  "enum": [
+    "Due date",
+    "Hold Expiration",
+    "Request Expiration"
```
Comment (kostyantyn-kh) [inline ramls/scheduled-notice-triggering-event.json]:
```suggestion
    "Request expiration"
```

Comment (dmytrotur) [reply ramls/scheduled-notice-triggering-event.json]: Done

File: ramls/patron-notice-policy.json (hunk 2 above)
Comment (dmytrotur) [reply ramls/patron-notice-policy.json]: Done

File: descriptors/ModuleDescriptor-template.json (hunk 1 above)
Comment (dmytrotur) [reply descriptors/ModuleDescriptor-template.json]: Actually, this pull request contains changes from https://github.com/folio-org/mod-circulation-storage/pull/186, where the version was changed from `0.1` to `0.2`. And this PR introduces new changes in the schema.

**Pull request closed without merging**